### PR TITLE
New Feature Flag Tracking page in RUM

### PR DESCRIPTION
**Pull request by meghanlo**
### What does this PR do?
* New page for Feature Flag Tracking in RUM
* Adding menu item for RUM

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by drichards-87** [comment]
Thanks for submitting this PR. I've created DOCS-4967 for the review. We'll get it assigned to someone this week.

File: content/en/real_user_monitoring/feature_flag_tracking/_index.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+- [Split][5]
+- [Flagsmith][6]
+
+Feature flags will show up in the context of events where they are evaluated, meaning they should show up on the views that the feature flag code logic is run on.
+
+## View your Feature Flags
+
+Once you have set up your feature flag data collection, navigate to the [**Feature Flags**][7] tab within RUM or click the [**Feature Flags**][7] tab under **UX Monitoring**
+
+From this view you can investigate any questions you have about your feature flag’s health and usage.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/feature_flag_tracking/_index.md:46]:
```suggestion
From this view, you can investigate any questions you have about your feature flag’s health and usage.
```

File: content/en/real_user_monitoring/feature_flag_tracking/_index.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+- Monitor the number of users experiencing each variant and see summary statistics of your feature flag
+- Check the status of your feature flag to see if there are any that can be removed for code clean up
+- View which pages your feature flags are being evaluated on
+
+{{< img src="real_user_monitoring/feature_flag_tracking/feature-flag-list.png" alt="View a list of your feature flags to investigate any questions you have about your feature flag’s health and usage" style="width:90%;" >}}
+
+
+### Search and filter
+Search and filter your feature flags by typing in the search bar. You can also use the faceted search, to narrow down, broaden, and shift your focus on subsets of feature flags you are interested in.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/feature_flag_tracking/_index.md:55]:
```suggestion
Search and filter your feature flags by typing in the search bar. You can also use the faceted search to narrow down, broaden, or shift your focus on subsets of feature flags you are interested in.
```

File: content/en/real_user_monitoring/feature_flag_tracking/_index.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+{{< img src="real_user_monitoring/feature_flag_tracking/feature-flag-list-search-filter.png" alt="Feature Flag list search bar and filtering" style="width:90%;" >}}
+
+### Feature Flag Status
+There are two possible feature flag statuses:
+- **Inactive**: For the past 2 weeks there have only been feature flag evaluations for your control variant
```
Comment (rtrieu) [inline content/en/real_user_monitoring/feature_flag_tracking/_index.md:61]:
```suggestion
- **Inactive**: For the past 2 weeks, there have only been feature flag evaluations for your control variant
```

File: content/en/real_user_monitoring/feature_flag_tracking/_index.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+- **Out to 100%**: For the past 2 weeks there have only been feature flag evaluations for one of your non-control variants
```
Comment (rtrieu) [inline content/en/real_user_monitoring/feature_flag_tracking/_index.md:62]:
```suggestion
- **Out to 100%**: For the past 2 weeks, there have only been feature flag evaluations for one of your non-control variants
```

File: content/en/real_user_monitoring/feature_flag_tracking/_index.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+## Analyze your feature flags
+To get more details about the health and performance of your feature flag, you can click on the flag in the list to navigate to a dedicated feature flag analysis dashboard. The Feature Flag analysis dashboard provides an overview of the performance of your feature flag, showing you information about user sessions, changes in your Core Web Vitals, and error rates.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/feature_flag_tracking/_index.md:65]:
```suggestion
To get more details about the health and performance of your feature flag, you can click the flag in the list to navigate to a dedicated feature flag analysis dashboard. The Feature Flag analysis dashboard provides an overview of the performance of your feature flag, displaying information about user sessions, changes in your Core Web Vitals, and error rates. 
```

File: content/en/real_user_monitoring/feature_flag_tracking/_index.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+### Feature Flag Status
+There are two possible feature flag statuses:
+- **Inactive**: For the past 2 weeks there have only been feature flag evaluations for your control variant
+- **Out to 100%**: For the past 2 weeks there have only been feature flag evaluations for one of your non-control variants
+
+## Analyze your feature flags
+To get more details about the health and performance of your feature flag, you can click on the flag in the list to navigate to a dedicated feature flag analysis dashboard. The Feature Flag analysis dashboard provides an overview of the performance of your feature flag, showing you information about user sessions, changes in your Core Web Vitals, and error rates. 
+
+These out of the box graphs are aggregated across your flag variants, making it easy to spot problems in your feature releases before they turn into serious issues. This dashboard provides an easy way to monitor your feature releases and allows you to quickly roll back as soon as you spot an issue so you can avoid negative user experiences.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/feature_flag_tracking/_index.md:67]:
```suggestion
These out-of-the-box graphs are aggregated across your flag variants, making it easy to spot problems in your feature releases before they turn into serious issues. This dashboard provides an easy way to monitor your feature releases and allows you to quickly roll back as soon as you spot an issue so you can avoid negative user experiences. 
```

File: content/en/real_user_monitoring/feature_flag_tracking/_index.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+The **Performance** tab allows you to understand if one of your feature flag variants have caused poor performance. You can view your Core Web Vitals and loading time for each variant to determine if one of your variants may be causing a negative impact on your application's performance.
+
+### Build custom views from Feature Flag data using the RUM Explorer
+Search through all the data collected by RUM in the [RUM Explorer][8] to surface trends on feature flags, analyze patterns with greater context, or export them into [dashboards][9] and [monitors][10]. 
+
+You can search your Sessions, Views, or Errors in the RUM Explorer, with the `@feature_flags.{flag_name}` attribute to scope down and focus on events where users were shown a specific user experience.
+
+You can compare important metrics to you and your teams by grouping your query by `@feature_flags.{flag_name}`. For example, if we want to understand how our new checkout flow is affecting the conversion rate from our checkout page to users making a purchase, we can add a "Group by" on our conversion rate graph.
```
Comment (rtrieu) [inline content/en/real_user_monitoring/feature_flag_tracking/_index.md:83]:
```suggestion
You can compare important metrics to you and your teams by grouping your query by `@feature_flags.{flag_name}`. For example, if you want to understand how your new checkout flow is affecting the conversion rate from the checkout page to users making a purchase, you can add a "Group by" on the conversion rate graph.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some small suggestions, but otherwise looks good!

**Review by rtrieu — APPROVED** [review]
Looks good, thanks!

**Pull request merged**